acceptable/tests/test_build_doubles.py

```diff
@@ -12,10 +20,19 @@
     Contains
 )
 
+
 from acceptable import _build_doubles
 
 
-class ExtractSchemasFromSourceTests(TestCase):
+class BuildDoubleTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
```

Comment:
hmm `super()` will fail on Py2, move it to a decorator a la https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures

Reply:
argumentless super() works fine via future module

Reply:
Yet you seem to have converted away from argumentless `super()` elsewhere?